x-pack/plugins/security_solution/public/ai_insights/pages/empty_prompt/index.tsx

```diff
@@ -0,0 +1,120 @@
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import React, { useMemo } from 'react';
+
+import { useAssistantAvailability } from '../../../assistant/use_assistant_availability';
+import { AnimatedCounter } from './animated_counter';
+import * as i18n from './translations';
+
+interface Props {
+  alertsCount: number;
+  isDisabled?: boolean;
+  isLoading: boolean;
+  onGenerate: () => void;
+}
+
+const EmptyPromptComponent: React.FC<Props> = ({
+  alertsCount,
+  isLoading,
+  isDisabled = false,
+  onGenerate,
+}) => {
+  const { hasAssistantPrivilege } = useAssistantAvailability();
+  const title = useMemo(
+    () => (
+      <EuiFlexGroup
+        alignItems="center"
+        data-test-subj="emptyPromptTitleContainer"
+        direction="column"
+        gutterSize="none"
+      >
+        <EuiFlexItem data-test-subj="emptyPromptAvatar" grow={false}>
+          <AssistantAvatar size="m" />
+          <EuiSpacer size="m" />
+        </EuiFlexItem>
+
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup alignItems="center" direction="row" gutterSize="none">
+            <EuiFlexItem data-test-subj="emptyPromptAnimatedCounter" grow={false}>
+              <AnimatedCounter count={alertsCount} />
+            </EuiFlexItem>
+
+            <EuiFlexItem data-test-subj="emptyPromptAlertsWillBeAnalyzed" grow={false}>
+              <span>{i18n.ALERTS_WILL_BE_ANALYZED(alertsCount)}</span>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    ),
+    [alertsCount]
+  );
+
+  const body = useMemo(
+    () => (
+      <EuiFlexGroup
+        alignItems="center"
+        data-test-subj="bodyContainer"
+        direction="column"
+        gutterSize="none"
+      >
+        <EuiFlexItem grow={false}>
+          <EuiText color="subdued" data-test-subj="basedOnSelectedFiltersLabel">
+            {i18n.BASED_ON_SELECTED_FILTERS}
+          </EuiText>
+        </EuiFlexItem>
+
+        <EuiFlexItem grow={false}>
+          <EuiText color="subdued" data-test-subj="startGeneratingInsightsLabel">
+            {i18n.START_GENERATING_INSIGHTS}
+          </EuiText>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    ),
+    []
+  );
+
+  const actions = useMemo(
+    () => (
+      <EuiButton
+        color="primary"
+        data-test-subj="generate"
+        disabled={!hasAssistantPrivilege || isLoading || isDisabled}
+        onClick={onGenerate}
+      >
+        {i18n.GENERATE}
+      </EuiButton>
+    ),
+    [hasAssistantPrivilege, isDisabled, isLoading, onGenerate]
+  );
+
+  return (
+    <EuiFlexGroup alignItems="center" direction="column" gutterSize="none">
+      <EuiFlexItem data-test-subj="emptyPromptContainer" grow={false}>
+        <EuiEmptyPrompt actions={actions} body={body} title={title} />
+      </EuiFlexItem>
+
+      <EuiFlexItem grow={false}>
+        <EuiLink data-test-subj="learnMore" href="#" target="_blank">
+          {i18n.LEARN_MORE}
+        </EuiLink>
```

Comment:
I imagine this is a placeholder for the docs link once available?

Reply:
Yes. Applying a `disabled` style to it effects the process of creating screenshots for documentation, so it's enabled, but just a placeholder until the docs URL is available.